monero/test_support.go

```diff
@@ -85,14 +85,15 @@ func BackgroundMineBlocks(t *testing.T) {
 		cancelFunc()
 		wg.Wait()
 	})
+	// Lower the sleep duration used by WaitForBlock
+	blockSleepDuration = backgroundMineInterval / 3
```

Comment:
I realized that our 10 second sleep when we are checking for a new monero block was too long when running unit tests where we are now mining a little less than 2 blocks/sec. I changed `blockSleepDuration` from a constant to a variable, but the only place that overrides it is here in this code that doesn't get compiled into our production binaries.